test/e2e-v2/cases/log/mysql/sharding/docker-compose.yml

```
@@ -0,0 +1,101 @@
+# Licensed to the Apache Software Foundation (ASF) under one or more
+# contributor license agreements.  See the NOTICE file distributed with
+# this work for additional information regarding copyright ownership.
+# The ASF licenses this file to You under the Apache License, Version 2.0
+# (the "License"); you may not use this file except in compliance with
+# the License.  You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+version: '2.1'
+
+services:
+  sharding-proxy:
+    extends:
+      file: ../../../../script/docker-compose/base-compose.yml
+      service: sharding-proxy
+    networks:
+      - e2e
+    ports:
+      - 3307
+    depends_on:
+      mysql-1:
+        condition: service_healthy
+      mysql-2:
+        condition: service_healthy
+
+  mysql-1:
+    image: mysql/mysql-server:8.0.13
+    networks:
+      - e2e
+    ports:
+      - 3306
+    environment:
+      MYSQL_ROOT_PASSWORD: "root@1234"
+      MYSQL_DATABASE: "swtest"
+      MYSQL_ROOT_HOST: "%"
+    healthcheck:
+      test: ["CMD", "bash", "-c", "cat < /dev/null > /dev/tcp/127.0.0.1/3306"]
+      interval: 5s
+      timeout: 60s
+      retries: 120
+
+  mysql-2:
+    image: mysql/mysql-server:8.0.13
+    networks:
+      - e2e
+    ports:
+      - 3306
+    environment:
+      MYSQL_ROOT_PASSWORD: "root@1234"
+      MYSQL_DATABASE: "swtest"
+      MYSQL_ROOT_HOST: "%"
+    healthcheck:
+      test: ["CMD", "bash", "-c", "cat < /dev/null > /dev/tcp/127.0.0.1/3306"]
+      interval: 5s
+      timeout: 60s
+      retries: 120
+
```

Try to use yaml anchor to reduce duplications

```suggestion

  mysql-1: &mysql
    image: mysql/mysql-server:8.0.13
    networks:
      - e2e
    ports:
      - 3306
    environment:
      MYSQL_ROOT_PASSWORD: "root@1234"
      MYSQL_DATABASE: "swtest"
      MYSQL_ROOT_HOST: "%"
    healthcheck:
      test: ["CMD", "bash", "-c", "cat < /dev/null > /dev/tcp/127.0.0.1/3306"]
      interval: 5s
      timeout: 60s
      retries: 120

  mysql-2:
    <<: *mysql
```